### ISSUE TITLE
Restart to load old channel messages should require confirmation

### ISSUE THREAD

**Issue by holmesworcester** [ZbayApp/zbay#471]
Context: when a user clicks "restart" to load older messages on a channel she has just joined, restart begins so fast it's almost jarring. 

We should show a modal asking users if they wish to restart. 

e.g. "Re-sync now? It will take 1-2 hours."

@NorbertBodziony is there a small modal that will work for this? Also, will rescan happen the next time they restart? Or only if they click restart now? (The latter is what we want, I think)

**Comment by NorbertBodziony** [comment]
Yes we already got small modal just for confirmation i will add it.
